src/components/predictive-layer/predictive-layer.vue

```diff
@@ -35,6 +35,14 @@
             : 'x-row__item--span-5 x-padding--right-00'
         ]"
       >
+        <BaseIdModalOpen
+          v-if="$x.device === 'mobile'"
+          modalId="my-history-aside"
+          class="x-button--ghost x-self-end x-padding--right-03"
```

Comment:
The button icon is a bit misaligned due to an invisible border

```suggestion
          class="x-button--ghost x-self-end x-padding--right-03 x-border-width--00"
```

Reply:
good catch!

Reply:
good catch!